examples/hello-world/main.cpp

```diff
@@ -545,18 +539,17 @@ void innerMain(ApplicationContext* context)
 
     if (SLANG_FAILED(app.initialize()))
     {
-        return exitApplication(context, 1);
+        return -1;
     }
 
-    while(dispatchEvents(context))
-    {
-        app.renderFrame();
-    }
+    platform::Application::run(app.gWindow);
 
     app.finalize();
+
+    return 0;
 }
 
 // This macro instantiates an appropriate main function to
 // invoke the `innerMain` above.
 //
-GFX_UI_MAIN(innerMain)
+PLATFORM_UI_MAIN(innerMain)
```

Comment:
Should this be `SLANG_PLATFORM_UI_MAIN` so that we always have the `SLANG_` prefix on our code, other than `gfx`?